files/collaborative_document_online.md

```diff
@@ -36,11 +36,13 @@ All content is publicly available under the Creative Commons Attribution License
 
 ## 🙋Getting help
 
-To ask a question, type `/hand` in the chat window.
+To ask a question, raise your hand in zoom. Click on the icon labeled "Reactions" in the toolbar on the bottom center of your screen,
+then click the button 'Raise Hand ✋'. For urgent questions, just unmute and speak up!
 
-To get help, type `/help` in the chat window.
+You can also ask questions or ask for help in the chat window and helpers will try to help you.
```

Comment:
This is what we actually do in practice

Reply:
Not every zoom client has a "raise hand" option this is why '/hand' in the chat, I think we can keep both then.

Reply:
Check